Runtime segfault

After cf862225d7a4a0d4ff08387c12bcf8ddcde4137e, I'm unable to run the server.

```sh
api on  dev via 🐹 v1.20.6 
❯ make build && ./out/api server
bash scripts/build
+ [[ -d out ]]
+ rm -rf out
+ binaries=(envsubst api)
+ for binary in "${binaries[@]}"
+ CGO_ENABLED=0
+ GOOS=linux
+ GOARCH=amd64
+ go build -o out/envsubst cmd/envsubst/main.go
+ for binary in "${binaries[@]}"
+ CGO_ENABLED=0
+ GOOS=linux
+ GOARCH=amd64
+ go build -o out/api cmd/api/main.go
+ static_files=(boundaries.json init.sh config.yaml.example)
+ static_dirs=(static docs)
+ for file in "${static_files[@]}"
+ cp boundaries.json out/boundaries.json
+ for file in "${static_files[@]}"
+ cp init.sh out/init.sh
+ for file in "${static_files[@]}"
+ cp config.yaml.example out/config.yaml.example
+ for dir in "${static_dirs[@]}"
+ cp -Rp static out/static
+ for dir in "${static_dirs[@]}"
+ cp -Rp docs out/docs
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x1e0 pc=0x10a07e4]

goroutine 1 [running]:
github.com/adh-partnership/api/internal/v1/router.init.0()
        /home/matt/code/denver_artcc/api/internal/v1/router/router.go:36 +0x1e4
```

Same for `go build cmd/api && ./out/api server` and `go run cmd/api/main.go server`. I tried deleting my project directory and recloning without success.

--- a/pkg/config/config.go
+++ b/pkg/config/config.go
@@ -4,10 +4,20 @@ import (
 	"os"
 	"strings"
 
+	"dario.cat/mergo"
 	"sigs.k8s.io/yaml"
 )
 
 var Cfg *Config
+
+// Primarily used to define Features and disable them by default
+// should facilities not configure them
+// We may extend this later with other defaults
+var defaultConfig = &Config{
+	Features: ConfigFeatures{
+		StaffingRequest: false,
+	},
+}
 
 func ParseConfig(file string) (*Config, error) {
 	// Read config file disk
@@ -23,6 +33,13 @@ func ParseConfig(file string) (*Config, error) {
 	}
 
 	sanitizeConfig(cfg)
+
+	// Merge in default, which will only "add" undefined values
+	// We use this to set feature flags to disabled by default
+	err = mergo.Merge(cfg, defaultConfig)
+	if err != nil {
+		return nil, err
+	}
 
 	return cfg, nil
 }

